jena-cmds/src/main/java/riotcmd/utf8.java

```diff
@@ -60,7 +60,8 @@ public static void main(String... args) {
                     } else
                         colNum++ ;
                     if ( !Character.isDefined(ch) )
-                        throw new AtlasException(String.format("No such codepoint: 0x%04X", ch)) ;
+                        //throw new AtlasException(String.format("No such codepoint: 0x%04X", ch)) ;
```

Comment:
Delete? :point_up:

Reply:
Done (this gives the test of being within the unicode version supported, not just any UTF-8 sequence).